### PR TITLE
Add test for DocumentAndElementEventHandlers in SVG

### PR DESCRIPTION
**Pull request by dstorey**
Based on html/webappapis/scripting/events/event-handler-all-document-element-events.html but for cut/copy/paste and only testing SVGElement, and using SVG template

This is part of ACTION-3867

### CODE REVIEW

**Comment by dstorey** [comment]
This is not in the spec yet. This adds tests before we commit the change to allow DocumentAndElementEventHandlers on SVGElement. This mixes in oncopy/cut/paste.

Firefox currently allows them on SVGElement (but fails tests). Chrome and Safari implement them on Element (which fails for HTML where they suppose to be on HTMLElement). Edge doesn't currently support them on SVGElement, but I have a PR close to sign off to add them.

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/26677)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/360089232)

**Comment by dstorey** [comment]
These now pass in Edge :)

**Comment by Ms2ger** [comment]
@smaug---- I think this exposes a bug in `IsEventAttributeNameInternal` for SVG; Gecko is not compiling the content attributes.

File: svg/types/scripted/event-handler-all-document-element-events.svg [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+     xmlns:h="http://www.w3.org/1999/xhtml">
+    <title>DocumentAndElementEventHandlers</title>
+    <metadata>
+        <h:link rel="help" href="https://svgwg.org/svg2-draft/types.html#InterfaceSVGElement"/>
+    </metadata>
+    <h:script src="/resources/testharness.js"/>
+    <h:script src="/resources/testharnessreport.js"/>
+    <script><![CDATA[
+        "use strict";
+        setup({ explicit_done: true });
```
Comment (Ms2ger) [inline svg/types/scripted/event-handler-all-document-element-events.svg]: There does not seem to be a need for this. Please remove it (and the `done()` call). I'd be happy to get the PR merged some time next week if you don't have time, though.

**Review by Ms2ger — APPROVED** [review]

**Comment by smaug----** [comment]
@jwatt

**Pull request merged**